src/System.Net.Security/tests/FunctionalTests/SslStreamSystemDefaultsTest.cs

```diff
@@ -95,6 +91,32 @@ public TestBase()
                 }
             }
 
+            private bool ServerCertCallback(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors)
+            {
+                switch (sslPolicyErrors)
+                {
+                    case SslPolicyErrors.None:
+                    case SslPolicyErrors.RemoteCertificateChainErrors:
+                    case SslPolicyErrors.RemoteCertificateNameMismatch:
+                        return true;
+                    case SslPolicyErrors.RemoteCertificateNotAvailable:
+                        if (PlatformDetection.IsWindows7 && !Capability.IsTrustedRootCertificateInstalled())
```

Comment:
If the problem is that the root cert isn't installed, that feels like it should be a chain error, not certificate not available...